### PR TITLE
Currency converter

### PR DESCRIPTION
**Pull request by Mahmoudmohamedalbanhawi**

### CODE REVIEW

File: src/app/Abstract/_variables.scss
Comment (MostafaAshrafMohamedKamal) [inline src/app/Abstract/_variables.scss:1]: must move _variable.scss to assets/scss/utils

File: src/app/CurrencySection/currency-section.component.scss
Comment (MostafaAshrafMohamedKamal) [inline src/app/CurrencySection/currency-section.component.scss:1]: move all shared scss from style.scss and add it to assets/scss/shared

File: src/app/CurrencySection/currency-section.component.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@import '../Abstract/variables';
+.container {
+    position: relative;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    background-image: url(../../assets/images/5171868.jpg);
```
Comment (MostafaAshrafMohamedKamal) [inline src/app/CurrencySection/currency-section.component.scss:5]: url(../../assets/images/5171868.jpg); It's not a correct way to get images of files from the assets folder.

File: src/app/CurrencySection/currency-section.component.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import '../Abstract/variables';
+.container {
+    position: relative;
```
Comment (MostafaAshrafMohamedKamal) [inline src/app/CurrencySection/currency-section.component.scss]: use bootstrap classes

File: src/app/CurrencySection/currency.ts
Comment (MostafaAshrafMohamedKamal) [inline src/app/CurrencySection/currency.ts:1]: Make a shared/interface folder to hold all shared files and move the ICurrency interface to it.

File: src/app/Navbar/Navbar.component.html
Comment (MostafaAshrafMohamedKamal) [inline src/app/Navbar/Navbar.component.html:1]: not match design (colors and  icons )

File: src/app/CurrencySection/currency-section.component.html
Comment (MostafaAshrafMohamedKamal) [inline src/app/CurrencySection/currency-section.component.html:1]: not match design ( container padding, button size and shadow )

File: src/app/CurrencySection/currency-section.component.ts
Comment (MostafaAshrafMohamedKamal) [inline src/app/CurrencySection/currency-section.component.ts:1]: should handle if the from and to are the same and if clicked when the amount is empty.

File: src/app/app.component.html
Comment (MostafaAshrafMohamedKamal) [inline src/app/app.component.html:1]: delete unused images from assets/images

File: src/app/Abstract/_variables.scss
Comment (Mahmoudmohamedalbanhawi) [reply src/app/Abstract/_variables.scss:1]: done

File: src/app/CurrencySection/currency-section.component.scss
Comment (Mahmoudmohamedalbanhawi) [reply src/app/CurrencySection/currency-section.component.scss:1]: done

File: src/app/CurrencySection/currency-section.component.scss (hunk 1 above)
Comment (Mahmoudmohamedalbanhawi) [reply src/app/CurrencySection/currency-section.component.scss:5]: done

File: src/app/CurrencySection/currency-section.component.scss (hunk 2 above)
Comment (Mahmoudmohamedalbanhawi) [reply src/app/CurrencySection/currency-section.component.scss]: done

File: src/app/CurrencySection/currency.ts
Comment (Mahmoudmohamedalbanhawi) [reply src/app/CurrencySection/currency.ts:1]: done

File: src/app/Navbar/Navbar.component.html
Comment (Mahmoudmohamedalbanhawi) [reply src/app/Navbar/Navbar.component.html:1]: i tried to change as much as i can

File: src/app/CurrencySection/currency-section.component.html
Comment (Mahmoudmohamedalbanhawi) [reply src/app/CurrencySection/currency-section.component.html:1]: i tried to change as much as i can

File: src/app/CurrencySection/currency-section.component.ts
Comment (Mahmoudmohamedalbanhawi) [reply src/app/CurrencySection/currency-section.component.ts:1]: finished

File: src/app/app.component.html
Comment (Mahmoudmohamedalbanhawi) [reply src/app/app.component.html:1]: done

**Pull request merged**